ecommerce-app/src/pages/api/categories.js

```diff
@@ -0,0 +1,6 @@
+// pages/api/categories.js
+import data from "../../db.json";
```

Comment:
We are directly importing data from db.json which is not right approach. We should use `json-server` instead.